lib/services/messenger/translators/front.ts

```diff
@@ -127,6 +128,38 @@ export class FrontTranslator extends TranslatorScaffold implements Translator {
 		return Promise.resolve(_.reverse(_.map(response._results, (comment) => comment.body )));
 	}
 
+	/**
+	 * Converts a response into the generic format.
+	 * @param _message  The initial message that prompted this action.
+	 * @param response  The response from the SDK.
+	 * @returns         Promise that resolve to the thread details.
+	 */
+	public static convertListWhispersResponse(
+		metadataConfig: MetadataConfiguration, _message: TransmitInformation, response: ConversationComments
+	): Promise<TranslatorMetadata[]> {
+		const rawComments = _.reverse(_.map(response._results, (comment) => comment.body ));
+		const comments = _.map(rawComments, (commentObject) => {
+			return FrontTranslator.extractMetadata(commentObject, MetadataEncoding.HiddenMD, metadataConfig);
+		});
+		return Promise.resolve(comments);
+	}
+
+	/**
+	 * Converts a response into the generic format.
+	 * @param _message  The initial message that prompted this action.
+	 * @param response  The response from the SDK.
+	 * @returns         Promise that resolve to the thread details.
+	 */
+	public static convertListRepliesResponse(
+		metadataConfig: MetadataConfiguration, _message: TransmitInformation, response: ConversationComments
+	): Promise<TranslatorMetadata[]> {
+		const rawComments = _.reverse(_.map(response._results, (comment) => comment.body ));
+		const comments = _.map(rawComments, (commentObject) => {
```

Comment:
```typescript
return Promise.resolve(_.map());
```
And same for any other instances in the code which make creating an extra variable probably unnecessary.